docs/testing-plan.md

```diff
@@ -0,0 +1,130 @@
+# Goal of testing plan
+Manual testing is expensive and we want to create a testing plan to focus on critical parts of the software while automated tests are being developed. 
+
+Some manual tests may be expensive to automate but we should aim to limit this list as much as possible and create as much test coverage to be confident in our deploys.
+
+When testing manual testing we need to keep in mind still both unit tests and integration tests.
+
+Unit tests: individual units or components of a software are tested. 
+Example: A button. Does it look like the mock up? Does it click event work as expected? 
```

Comment:
```suggestion
Example: A button. Does it look like the mock up? Does the click event work as expected? 
```

Reply:
Also, we should decide if we want to capitalize after the colon or not. Im not the best when it comes to grammar so Im not sure what the rule is 😅. In the other examples you arent capitalizing after the colon.